lib/jnpr/junos/utils/fs.py

```diff
@@ -251,7 +251,7 @@ def storage_usage(self):
         """
         rsp = self._dev.rpc.get_system_storage()
 
-        _name = lambda fs: fs.findtext('filesystem-name').strip()
+        def _name(fs): return fs.findtext('filesystem-name').strip()
```

Comment:
@rsmekala Can you explain me this change

Reply:
It was auto-edited by autopep8 package.

Please find below excerpt from [PEP8 styling guide](https://www.python.org/dev/peps/pep-0008/):
```
Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier.

Yes:

def f(x): return 2*x
No:

f = lambda x: 2*x
The first form means that the name of the resulting function object is specifically 
'f' instead the generic '<lambda>'. This is more useful for tracebacks and string 
representations in general. The use of the assignment statement eliminates the sole
benefit a lambda expression can offer over an explicit def statement (i.e. that it 
can be embedded inside a larger expression)
```